src/controllers/transfers.js

```diff
@@ -65,12 +66,13 @@ exports.getResource = function * fetch () {
  * @apiVersion 1.0.0
  *
  * @apiDescription Use this to get a signed receipt containing only the id of
- *   transfer and its state.
+ *   transfer and its state. It functions even if the transfer doesn't exist yet.
```

Comment:
Might be worth explicitly saying that if the transfer does not exist yet it will have the state set to `nonexistent`